.travis.yml

```diff
@@ -27,24 +27,40 @@ jobs:
 
     # Primary platforms
 
+    # pod lib lint to check build and warnings
```

Comment:
The stages are ordered to give the fastest possible feedback. Whenever I've run `pod lib lint` the process has been fairly slow. If these stages take longer than the regular builds `METHOD=xcodebuild` I suggest moving the lint stages down.

Reply:
I suspect that pod lib lint might fail faster, but will push it down until we have more data.